src/com/palmergames/bukkit/towny/TownyUniverse.java

```diff
@@ -142,6 +144,16 @@ boolean loadSettings() {
             return false;
         }
         
+        Version lastRunVersion = new Version(TownySettings.getLastRunVersion(Towny.getPlugin().getVersion()));
+        Version curVersion = new Version(Towny.getPlugin().getVersion());
+        
+        // Only migrate if the user just updated.
+        if (!lastRunVersion.equals(curVersion)) {
+			System.out.println("[Towny] Performing Config Migrations...");
```

Comment:
Use the logger here

Reply:
I prefer not using the logger in teh startup sequence, because most of it doesn't and if some does use it, it jumps all over with different indentation. 

The other one above can probably be a debug message, I glazed over that one.